dist/common/scripts/scylla_io_setup

```diff
@@ -142,6 +142,39 @@ if __name__ == "__main__":
             yaml.dump({ "disks": [ disk_properties ] }, properties_file,  default_flow_style=False)
             ioconf = open(etcdir() + "/scylla.d/io.conf", "w")
             ioconf.write("SEASTAR_IO=\"--io-properties-file={}\"\n".format(properties_file.name))
+        elif gcp_instance().is_gce_instance() and (is_debian_variant() or is_redhat_variant()):
+            idata = gcp_instance()
 
+            if idata.is_recommended_instance():
+                disk_properties = {}
+                disk_properties["mountpoint"] = datadir()
+                nr_disks = idata.nvmeDiskCount
+                if nr_disks >= 1 & nr_disks < 4:
+                    disk_properties["read_iops"] = 179861 * nr_disks
+                    disk_properties["read_bandwidth"] = 737304512 * nr_disks
+                    disk_properties["write_iops"] = 99963 * nr_disks
+                    disk_properties["write_bandwidth"] = 409628288 * nr_disks
+                elif nr_disks >= 4 & nr_disks <= 8:
+                    disk_properties["read_iops"] = 721719
+                    disk_properties["read_bandwidth"] = 2951769344
+                    disk_properties["write_iops"] = 400777
+                    disk_properties["write_bandwidth"] = 1592197376
+                elif nr_disks == "16":
+                    disk_properties["read_iops"] = 1682373
+                    disk_properties["read_bandwidth"] = 4521251328
+                    disk_properties["write_iops"] = 910985
+                    disk_properties["write_bandwidth"] = 2759452672
+                elif nr_disks == "24":
+                    disk_properties["read_iops"] = 2502141
+                    disk_properties["read_bandwidth"] = 5921532416
+                    disk_properties["write_iops"] = 1290502
+                    disk_properties["write_bandwidth"] = 4663037952
+                properties_file = open(etcdir() + "/scylla.d/io_properties.yaml", "w")
```

Comment:
Is it possible to have nr_disks = 19?

In any case we need to detect it (with an extra else clause), not just fall through and write a partial file.

Reply:
not possible, GCP blocks it